### ISSUE TITLE
It crashes when playing streams on Fedora 24

### ISSUE THREAD

**Issue by michaltrzcinski** [gnumdk/lollypop#739]
Lollypop plays stream ~1s and crashes immediatelly.

```
$ python3 --version
Python 3.5.1

$ lollypop -d
No module named 'wikipedia'
Advanced artist informations disabled
$ sudo pip3 install wikipedia
Player::_on_stream_start(): mms://stream.polskieradio.pl/program3_wma10
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/lollypop/mpris.py", line 385, in __on_current_changed
    self.__update_metadata()
  File "/usr/lib/python3.5/site-packages/lollypop/mpris.py", line 343, in __update_metadata
    Lp().player.current_track.duration * 1000000)
TypeError: unsupported operand type(s) for *: 'NoneType' and 'int'

```

**Comment by gnumdk** [comment]
Are you using last lollypop version? 
#736 

duration should not be None anymore...

**Comment by michaltrzcinski** [comment]
I'm using 0.9.203. See now it was fixed in 0.9.204.
Fedora Copr is down and this is why we don't have latest package version.

Thank you. Great app.
